shared/profile/index.js

```diff
@@ -26,6 +39,19 @@ class Profile extends Component<void, Props, void> {
 export default connect(
   state => ({
     username: state.config.username,
+    trackers: state.tracker.trackers,
+  }),
+  dispatch => ({
+    refresh: username => dispatch(getProfile(username)),
   }),
-  dispatch => ({})
+  (stateProps, dispatchProps, ownProps) => {
+    // const username = ownProps.username || stateProps.username
+
+    return {
+      ...ownProps,
+      ...stateProps.trackers[username],
+      ...dispatchProps,
```

Comment:
Is it necessary to merge `dispatchProps` in here? Does it contain anything beyond the `refresh` function defined above? I'm not sure if this can be cleaned up, but noting it in case it can.